tensorflow/python/eager/wrap_function.py

```diff
@@ -277,18 +311,26 @@ def _fetch_preprocesing_callback(f):
 
     fetches = nest.map_structure(_fetch_preprocesing_callback, fetches)
 
-    for f in flat_feeds + tensor_fetches + operation_fetches:
+    for f in flat_feeds + tensor_fetches + operation_fetches \
```

Comment:
Could you use parens for line continuation here? http://google.github.io/styleguide/pyguide.html#32-line-length

Reply:
Sure, no problem. I've made that change in my local branch.

Reply:
Changes pushed to my branch.